geth/node/manager.go

```diff
@@ -361,7 +368,7 @@ func (m *NodeManager) restartNode() (<-chan struct{}, error) {
 	<-nodeStopped
 	m.Lock()
 
-	return m.startNode(&prevConfig)
+	return m.startNode(&prevConfig, nil)
```

Comment:
Looks like we need here same options as for start node. Please add test case for this.